airflow/config_templates/config.yml

```diff
@@ -861,6 +861,13 @@
       type: string
       example: ~
       default: "-"
+    - name: access_logformat
+      description: |
+        Access log format for gunicorn webserver
+      version_added: ~
+      type: string
+      example: ~
+      default: "%%(h)s %%(l)s %%(u)s %%(t)s \"%%(r)s\" %%(s)s %%(b)s \"%%(f)s\" \"%%(a)s\""
```

Comment:
Should we define default value? If we pass an empty value, the default value for gunicorn will not be used?

Reply:
Made changes, so that we don't need to specify default config. If empty value is passed, default value for gunicorn will be used.